docs/v13/documentation/make-first-prototype/let-user-change-answers.md.njk

```diff
@@ -7,8 +7,10 @@ caption: Build a basic prototype
 
 Make the **Change** links on the ‘Check answers’ page work by adding the right links.
 
-1. In the `<a>` tag under {% raw %}`{{ data['how-many-balls'] }}`{% endraw %}, change the href attribute from `#` to `/juggling-balls`
-2. In the `<a>` tag under {% raw %}`{{ data['most-impressive-trick'] }}`{% endraw %}, change the href attribute from `#` to `/juggling-trick`
+1. Find the {% raw %}`{{ govukSummaryList({`{% endraw %} component with 'Number of balls you can juggle' inside.
+2. Change the `href` from `#` to `/juggling-balls`
+3. Find the {% raw %}`{{ govukSummaryList({`{% endraw %} component with 'Your most impressive juggling trick' inside.
```

Comment:
I think this makes it sound like there are 2 `govukSummaryList`s when its just one with multiple rows